ansible/roles/cri/tasks/containerd.yml

```diff
@@ -27,7 +27,7 @@
 # Hardcoding due to the 20.04 ubuntu dist not currently available.
 - name: 'debian : add apt repository for stable version'
   apt_repository:
-    repo: deb [arch=arm64] https://download.docker.com/linux/ubuntu xenial stable
+    repo: deb https://download.docker.com/linux/ubuntu xenial stable
```

Comment:
We needed to hardcode `xenial` however, we should check to see if that is also required as we could use the previous logic to check the Ubuntu version being used.